purchase_allowed_product/README.rst

```diff
@@ -7,7 +7,7 @@ Purchase and Invoice Allowed Product
    !! This file is generated by oca-gen-addon-readme !!
    !! changes will be overwritten.                   !!
    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-   !! source digest: sha256:dad52af1ba2e3221412aeab43a4aac234fb45a36a9089fdf4b6f00db38f58ae9
+   !! source digest: sha256:d32423644e8e6dc1fb71183768911fb2232be0c7e2d1630a2356f526b6ac8ce5
```

Comment:
Can you clean up this change? thanks! @renda-dev

Reply:
it is changed by ocabot afaik

Reply:
@HviorForgeFlow could you update your review?

Reply:
@HviorForgeFlow gentle reminder

Reply:
@HviorForgeFlow gentle reminder